shell/platform/linux/public/flutter_linux/fl_view.h

```diff
@@ -69,6 +69,15 @@ FlView* fl_view_new_for_engine(FlEngine* engine);
  */
 FlEngine* fl_view_get_engine(FlView* view);
 
+/**
+ * fl_view_set_background_color:
+ * @view: an #FlView.
+ * @color: a background color.
+ *
+ * Set the background color for Flutter (defaults to black).
```

Comment:
Should we mention that this copies the color and that the caller can deallocate their color after calling this? Or is this a common pattern in GTK that a user will understand?

Reply:
This is a common pattern - passing a pointer assumes it will be copied if required unless explicitly stated.